Makefile

```diff
@@ -23,7 +23,7 @@
 #   CHECK_LEAKS      - test for memory leaks (debugging mode only)
 #   VZ3              - compile with Z3
 
-COMMON_FLAGS = -DVTIME_PROFILING=0 -DVMINI_GMP
+COMMON_FLAGS = -DVTIME_PROFILING=0
 
 DBG_FLAGS = $(COMMON_FLAGS) -g  -DVDEBUG=1 -DCHECK_LEAKS=0 # debugging for spider
 REL_FLAGS = $(COMMON_FLAGS) -O3 -DVDEBUG=0 -DNDEBUG # no debugging
```

Comment:
There is the `NDebug` definition here, and it has been removed in the `CMakeLists.txt` I'm not sure what it does. Just wanted to check whether this was intentional?

Reply:
`NDEBUG` among other things means that `assert()` compiles to nothing - apparently it gets set by CMake on release builds.